src/stories/datadisplay/Avatar.stories.tsx

```diff
@@ -1,17 +1,22 @@
-import React from "react";
-import { Meta, Story } from "@storybook/react";
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
 import { Avatar, AvatarProps } from "../../components/Avatar";
 import pkg from "../../components/Avatar/package.json";
```

Comment:
I think your prettier or eslint config is doing something weird here. Half of the imports use single quotes and half of them use double quotes.

Reply:
Probably did not bother to check everywhere, will fix after merge conflicts are resolved.